packages/backend/src/services/ProjectService/ProjectService.mock.ts

```diff
@@ -160,6 +160,9 @@ export const expectedAllExploreSummary: SummaryExplore[] = [
         name: validExplore.name,
         label: validExplore.label,
         tags: validExplore.tags,
+        databaseName: validExplore.tables[validExplore.baseTable].database,
+        schemaName: validExplore.tables[validExplore.baseTable].schema,
```

Comment:
this is not needed, right ?

Reply:
@rephus  this is for TableInfo component, which is not being used at this moment. I added a loom where I explain why I kept TableInfo/LineageViews